appmap/django.py

```diff
@@ -140,16 +142,27 @@ def request_params(request):
     return values_dict(params.lists())
 
 def get_resolved_match(path_info, resolver):
-    # Look through the resolver's cache of reverse
-    # route mappings and find the regex that matches the current
-    # request. The route mappings are tuples of the form:
-    #   (format string, regex, args, kwargs)
-    regex = None
-    for _,values in resolver.reverse_dict.lists():
-        for _,regex,_,_ in values:
-            match = re.match(regex, path_info[1:])
-            if match:
-                return (regex, match)
+    resolver_match = resolver.resolve(path_info)
+    regex = resolver_match.route
```

Comment:
It's a bit confusing that this variable is called `regex`. Maybe rename to `route`?

Reply:
I guess. The unfortunate thing is that `resolver_match.route` *is* a regex sometimes, and the ultimate goal here is to make sure we have a regex that matches `path_info`.